src/views/htmlcontent/src/js/components/contextmenu.component.ts

```diff
@@ -83,7 +80,8 @@ export class ContextMenu implements OnInit {
         this.index = index;
         this.selection = selection;
         this.isDisabled = (selection.length > 1);
-        this.position = { x: x, y: y};
+        jQuery('.contextMenu').get(0).style.top = `${y.toString()}px`;
```

Comment:
Why did you remove the angular binding? its preferable over useing discord

Reply:
angular binding doesnt refresh the view because the main html is static